src/sentry/runner/commands/run.py

```diff
@@ -658,6 +658,18 @@ def handler(signum, frame):
     streamer.run()
 
 
+@run.command("billing-metrics-consumer")
+@log_options()
+@batching_kafka_options("billing-metrics-consumer")
```

Comment:
This decorator allows the client to specify `max-batch-size` and `max-batch-time-ms`.  That generally tells you how often you are supposed to commit.

Reply:
Thanks, now used by the strategy.